### ISSUE TITLE
first_day_of_week in getOrderMetrics only working with lowercase

### ISSUE THREAD

**Issue by graphicalx** [jlevers/selling-partner-api#120]
Hi,

Thanks for the great library!
I just set things up, and was trying the getOrderMetrics call with the default first 3 arguments when I got this error:
```
"errors": [
    {
      "code": "InvalidInput",
      "message": "{getOrderMetrics.arg7\u003dmust match \"(monday|sunday)\"}",
      "details": ""
    }
  ]
```

Interestingly if you change the default value of $first_day_of_week to `monday` (lowercase) it starts working. I wonder if that's a Selling Partner API bug, since they mention `Monday` explicitly.

Thanks

**Comment by jlevers** [comment]
I ran into this recently too -- I think it's an SP API bug. I'm going to update the model files I generate this library from to use the lowercase day names by default (Amazon's original models have them capitalized), which should solve the issue for the library. I'll close this issue when I push a release with that fix.

**Comment by jlevers** [comment]
Fixed in `v4.1.1`.
